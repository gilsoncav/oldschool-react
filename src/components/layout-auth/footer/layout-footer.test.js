import LayoutFooter from './layout-footer';

describe('LayoutFooter component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<LayoutFooter />);
    expect(wrapper).toMatchSnapshot();
  });
});
