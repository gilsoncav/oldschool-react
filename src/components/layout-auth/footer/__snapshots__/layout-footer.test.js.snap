// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutFooter component renders correctly 1`] = `
<footer
  className="ant-layout-footer"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <span>
    2019 - Kevin Aguiar -
     
    <a
      href="https://github.com/kevinfaguiar"
    >
      Github
    </a>
  </span>
</footer>
`;
